assets/queries/ansible/azure/storage_account_tls_version/metadata.json

```diff
@@ -0,0 +1,8 @@
+{
+  "id": "storage_account_tls_version",
+  "queryName": "Storage Account TLS Version",
```

Comment:
The query name should identify the vulnerability or fault, in this case, something like: "Storage Account Is Not Using Latest TLS Encryption Version"